### PR TITLE
Ws 042 home leaderboard

### PR DESCRIPTION
**Pull request by DeepashriDali**
Added leaderboard to the homepage
image

### CODE REVIEW

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  } else
+    return (
+      <>
+        <h1 className="home-leaderboard__title">Leaderboard Top 3</h1>
+        <section className="home-leaderboard__text-container">
+          <section className="home-leaderboard__text home-leaderboard__left">
+            <ul className="home-leaderboard__list-item">
+              <li>
+                <h5 className="home-leaderboard__list-item-heading">
+                  🥇 {topEntries[0].username} with a total of{" "}
```
Comment (TimLeach635) [inline frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:28]: Prettier has added its fun little things
```suggestion
                  🥇 {topEntries[0].username} with a total of
```

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                  {topEntries[0].count} sightings
+                </h5>
+              </li>
+              <li>
+                <h5 className="home-leaderboard__list-item-heading">
+                  🥈 {topEntries[1].username} with a total of{" "}
```
Comment (TimLeach635) [inline frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:34]: Remove `{" "}`

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                  {topEntries[1].count} sightings
+                </h5>
+              </li>
+              <li>
+                <h5 className="home-leaderboard__list-item-heading">
+                  🥉 {topEntries[2].username} with a total of{" "}
```
Comment (TimLeach635) [inline frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:40]: Remove `{" "}`

**Review by TimLeach635 — APPROVED** [review]
Just one tiny change before it's good to go!

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx (hunk 1 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:28]: Removed {" "}

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx (hunk 2 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:34]: Removed {" "}

File: frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx (hunk 3 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/homepageLeaderboard/homepageLeaderboard.tsx:40]: Removed {" "}

**Pull request merged**